www/docs/reactjs/getQueryKey.md

```diff
@@ -31,6 +31,17 @@ type QueryType = "query" | "infinite" | "any";
 // will match all ───────────────────────┘
 ```
 
+:::note
+
+The query type `any` will match all queries in the cache only if the `react query` method where it's used uses fuzzy matching.
+
+General rule of thumb:
+
+- functions with a singular in the name (getQueryData) operate on one query, so exact
+- functions with a plural in the name (invalidateQueries) operate on multiple queries, so they match fuzzily.
```

Comment:
ideally this'd be linked to rq docs but couldn't find any good not there. this is from TKDodo's resposne here: https://github.com/TanStack/query/issues/5111#issuecomment-1464864361

Reply:
maybe just link that comment :)

Reply:
good call